#30 create rds (postgre sql) module

TysonBeck opened the pull request.

TysonBeck committed:
#30 Initial commit for Postgres RDS
Duplicated RDS module and removed all unnecessary non Postgres resources

TysonBeck committed:
#30 Added defaults engine and port variables
Passed through staging_prefix var to instance sub-module

TysonBeck committed:
Merge branch 'master' into issue/30-Create-RDS-(PostgreSQL)-Module

jeremyodle commented on aws/rds-postgres/db.tf:
```diff
@@ -0,0 +1,516 @@
+  tags         = "${local.tags}"
+}
+
+module "db_parameter_group" {
```
I thought Postgres doesn't support parameter groups, is this still needed?

TysonBeck replied:
Postgres does support parameter groups. It does not support option groups.

jeremyodle committed:
#30 Set module name to `rds-postgres`

jeremyodle committed:
#30 Changed environment to `shared`

jeremyodle reviewed (changes requested):
Updated some code. Please pull from origin and validate Terraform plan again.

TysonBeck committed:
#30 Modified resource naming to use module prefix
Modified module prefix to work with or without a name provided
Removed MSSQL AD IAM role access
